framework/db/Command.php

```diff
@@ -780,6 +780,16 @@ public function execute()
 
             $this->pdoStatement->execute();
             $n = $this->pdoStatement->rowCount();
+            
+            // https://bugs.php.net/bug.php?id=61613
+            // Make sure that supported drivers throw an exception in case a statement of a multi-query failed
+            try {
+                while ($this->pdoStatement->nextRowSet()) {}
+            } catch (\Exception $e) {
+                if (strpos($e->getMessage(), 'SQLSTATE[IM001]') === false) {
```

Comment:
Hmm. But this is not documented anywhere, right (apart from the test case)? So it could change in a later PHP release. How about using `method_exists()` instead?